### PR TITLE
#2894 [Manager] Don't start a topic if it's failed in the creation phase

### PR DESCRIPTION
**Pull request by wu87988622**
### Checklist
- [x] [Review Contribution Guidelines](https://github.com/oharastream/ohara/blob/master/CONTRIBUTING.md)
- [x] Add Reviewers (see [Authors](https://github.com/oharastream/ohara#authors))
- [x] Add Assignees (of course, you are the assignee by default!)
- [x] DON'T set a Milestone
- [x] DON'T add any labels (PR labels will be added automatically by Jenkins)
- [x] Link to an issue(You can do so by using the required by keyword: required by #2894 )
- [ ] Add unit tests (or please explain why this PR is not worth having new tests)
- [x] Pass QA (Sometimes you may encounter an existing flaky test which would obstruct you from getting QA passed. If the failed tests are unrelated, you can add a comment in the PR thread)

### CODE REVIEW

File: ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesNewModal/WorkspacesNewModal.js [hunk 1]
```diff
@@ -418,7 +418,7 @@ const WorkerNewModal = props => {
     <Form
       onSubmit={onSubmit}
       initialValues={{}}
-      validate={validate}
+      validate={values => nameValidate({ values, key: 'name' })}
```
Comment (konekoya) [inline ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesNewModal/WorkspacesNewModal.js]: Does this handle multiple keys?

Comment (wu87988622) [reply ohara-manager/client/src/components/pages/WorkspacesPage/WorkspacesNewModal/WorkspacesNewModal.js]: ok I adjusted to support multiple keys

**Comment by konekoya** [comment]
@wu87988622 When creating a new topic, I'm getting the correct message, but the modal's loading state is not reset.

**Comment by wu87988622** [comment]
> @wu87988622 When creating a new topic, I'm getting the correct message, but the modal's loading state is not reset.

ok thanks

**Comment by wu87988622** [comment]
retry manager-e2e

**Pull request merged**